pkg/controller/kfservice/kfservice_controller.go

```diff
@@ -166,9 +165,9 @@ func (r *ReconcileService) updateStatus(before *kfservingv1alpha1.KFService, ksv
 		// to status with this stale state.
 
 	} else if err := r.Update(context.TODO(), after); err != nil {
-		log.Error(err, "Failed to update kfsvc status")
+		log.Error(err, "Failed to update KFService status")
```

Comment:
to be consistent with above to use full name, can we say `kfserving service` here?